engine/api/event/publish_application.go

```diff
@@ -117,3 +117,21 @@ func PublishApplicationKeyDelete(projKey string, app sdk.Application, k sdk.Appl
 	}
 	publishApplicationEvent(e, projKey, app.Name, u)
 }
+
+// PublishApplicationRepositoryAdd publishes an envet when adding a repository to an application
+func PublishApplicationRepositoryAdd(projKey string, app sdk.Application, u *sdk.User) {
+	e := sdk.EventApplicationRepositoryAdd{
+		VCSServer:  app.VCSServer,
+		Repository: app.RepositoryFullname,
+	}
+	publishApplicationEvent(e, projKey, app.Name, u)
+}
+
+// EventApplicationRepositoryDelete publishes an envet when deleting a repository to an application
```

Comment:
comment on exported function PublishApplicationRepositoryDelete should be of the form "PublishApplicationRepositoryDelete ..."